snyk/snyk
# feat: Pipfile support for `snyk fix`

#### What does this PR do?
Introduce `Pipfile` & `Pipfile.lock` fix support. 

#### Where should the reviewer start?
`packages/snyk-fix/src/plugins/python/get-handler-type.ts`

Found a small bug, which is fixed here: https://github.com/snyk/snyk/pull/1922

#### How should this be manually tested?
- install deps in a Pipfile project
- `snyk test` to see issues and which are fixable
- `snyk fix` to automatically fix the fixable issues.
#### Screenshots

![CleanShot 2021-05-17 at 13 40 49@2x](https://user-images.githubusercontent.com/2911613/118490140-88aa7200-b715-11eb-895f-b9d820b1947a.png)
